airflow/providers/databricks/hooks/databricks.py

```diff
@@ -404,6 +479,40 @@ def start_cluster(self, json: dict) -> None:
         """
         self._do_api_call(START_CLUSTER_ENDPOINT, json)
 
+    def activate_cluster(self, json: dict, polling: int, timeout: int | None = None) -> None:
+        """
+        Start the cluster, and wait for it to be ready.
+
+        :param json: json dictionary containing cluster specification.
+        :param polling: polling interval in seconds.
+        :param timeout: timeout in seconds. -1 means no timeout.
+        """
+        cluster_id = json['cluster_id']
```

Comment:
We always use double quotes in the project
```suggestion
        cluster_id = json["cluster_id"]
```